hw/top_earlgrey/data/top_earlgrey.sv.tpl

```diff
@@ -671,19 +671,19 @@ slice = str(alert_idx+w-1) + ":" + str(alert_idx)
   // Pinmux connections
   % if num_mio_outputs + num_mio_inouts != 0:
   assign mio_d2p = {
-    % for sig in top["pinmux"]["inouts"] + top["pinmux"]["outputs"]:
+    % for sig in list(reversed(top["pinmux"]["inouts"] + top["pinmux"]["outputs"])):
```

Comment:
@eunchan 
@msfschaffner 

i had to put in a reverse here because the software / hardware ordering were exactly reversed.
That's caused gpio read backs to fail and boot strap to fail also.